string motion_key
string filepath
---
bool success